core/arch/arm/plat-totalcompute/platform_config.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please put the SPDX line at the first line as a single comment. This comment applies to all files with a SPDX line.

While BSD 3 clause license is OK we're normally using BSD 2 clause. No big deal, but if it's the same to you please use BSD 2 clause instead.

Reply:
Fixed all files, moved SPDX line at the first line and changed to BSD 2 clause.